docs/release-notes/checkpoint-fs-server-download.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Improvements**
+
+-  Checkpoint downloads through the server now work with ``shared_fs`` types of
+   ``checkpoint_storage``.
```

Comment:
:orphan:

**Improvements**

-  Allow checkpoint downloads through the server for ``checkpoint_storage`` type ``shared_fs``.